cvat/apps/engine/serializers.py

```diff
@@ -2345,14 +2460,47 @@ def to_representation(self, instance):
         representation['consensus_enabled'] = self.get_consensus_enabled(instance)
         return representation
 
-
-class TaskWriteSerializer(WriteOnceMixin, serializers.ModelSerializer):
+@extend_schema_serializer(
+    examples=[
```

Comment:
Could you restore the original default example for regular field updates? Swagger only shows these new examples now.

Reply:
So, I can add a default example as it was, but it will include all fields, and I do not really think it's right to show different operations in one example.

Reply:
It was convenient to have an example with all the possible fields in swagger. I'm just asking to add 1 more example for updating regular fields.